src/UnitTestSharedLibrary/Utility.cs

```diff
@@ -58,7 +58,7 @@ public static A11yElement LoadA11yElementsFromJSON(string path)
         /// <returns></returns>
         public static void PopulateChildrenTests(A11yElement ke)
         {
-            ke.ScanResults.Items.ForEach(item => {
+            ke?.ScanResults?.Items?.ForEach(item => {
```

Comment:
The old code would throw NullReferenceException if ke or ke.ScanResults or ke.ScanResults.Items were null. Should be retain the throwing behavior for bad inputs, but map them to a flavor of ArgumentException?

Reply:
Fixed

Reply:
Do we want protection if ke.ScanItems == null or ke.ScanItems.Results == null?

---
In reply to: [311581411](https://github.com/microsoft/axe-windows/pull/130#discussion_r311581411) [](ancestors = 311581411)